Restructure landing page

beamandala opened the pull request:
Fixes #158

beamandala commented:
@axdyv Let me know what you think

axdyv commented:
i think theres too much space between the navbar and where the first components start. you could probably move everything up a little bit. Also what happened to peechi and some of the dots behind the slideshow component 😭

beamandala commented:
I think I did that because some of the bubbles were over the text and images. I'll have to look into the parallax component and move those around.

axdyv commented:
the top margin looks good! why is the parallax component so tweaky... 😭

beamandala commented:
@axdyv The positioning of the parallax images should be fixed now. Let me know if it looks good.

axdyv commented:
can you remove or make the dot that is next to peechi back a layer? It blocks the "of" in the University of Texas at Dallas in the first text bubble when you move your mouse in certain positions. Also can you move peechi back to where he is on the website/figma? While I see the vision behind putting it in the middle, I think it looks a little better where it was originally.

beamandala commented:
Moved the bubbles behind the home page layout so we won't have issues of them covering up text and moved peechi to the position it is in the figma design.

axdyv commented:
Waiting on Ruben to get the slideshow components onto the page now; Capitalize all the words in the bubbles and keep it consistent

axdyv reviewed (approved):
yea this fyaaaa
